iron-form.html

```diff
@@ -135,27 +158,25 @@
       */
 
       attached: function() {
-        this._nodeObserver = Polymer.dom(this.$.content).observeNodes(
```

Comment:
This observeNodes is needed so that you can use an `iron-form` imperatively (i.e. add a form manually). Also, it's distributed, which isn't necessarily ready in attached